frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -3,6 +3,10 @@ import { DEPLOYMENT_STRATEGY } from '../constants';
 import { OverviewItemAlerts, PodControllerOverviewItem } from './pod';
 import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
 
+export type ErrorWithMessage = {
```

Comment:
Wondering about a custom `Error` object type here, since it's not extending `Error`, is it right to call it an error?

Reply:
slack note:  maybe `APIError`.